openslides_backend/action/actions/vote/remove_user_id.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Dict
+
+from ....models.models import Vote
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+@register_action("vote.remove_user_id", internal=True)
+class VoteRemoveUserId(UpdateAction):
```

Comment:
Please rename this to `VoteAnonymize` for consistency

Reply:
Renamed into `VoteAnonymize`.